src/Acuminator/Acuminator.Tests/Acuminator.Tests.csproj

```diff
@@ -184,6 +184,10 @@
     <EmbeddedResource Include="Sources\BQL\Diagnostics\ArgumentsMismatch\SearchCall.cs" />
     <EmbeddedResource Include="Sources\Dac\PX1031 and PX1032\Diagnostics\DacWithMethodsUsage.cs" />
     <EmbeddedResource Include="Sources\Dac\PX1031 and PX1032\Diagnostics\DacExtensionWithMethodsUsage.cs" />
+    <EmbeddedResource Include="Sources\Dac\PX1040\Diagnostics\ConstructorInGraphExtension.cs" />
```

Comment:
@dropsonic I don't think graph extension is related to DAC, please add it to PXGraph folder or add new folder for graph extensions

Reply:
@dropsonic Notjing is added to HackathonDemo solution

Reply:
@SENya1990 that's my mistake, I'll move those files
I've added an example to the HackathonDemo solution, but it was not pushed because of *git* error. See my latest changes.